cvat-core/src/exceptions.ts

```diff
@@ -91,31 +91,6 @@ export class Exception extends Error {
             }),
         );
     }
-
-    async save(): Promise<void> {
```

Comment:
not used api, we use `events` api to save exceptions
`serverProxy.server.exception` even is not implemented